### ISSUE TITLE
About parameters --syncbn and --ngpus in the train.py

### ISSUE THREAD

**Issue by Mr-QingYe** [zhanghang1989/PyTorch-Encoding#316]
Hi，when I use the train.py to train EncNet_ResNet101s_VOC on the Pascal Voc dataset，I click the cmd link to get command。The command as follows:

# First finetuning COCO dataset pretrained model on augmented set
# You can also train from scratch on COCO by yourself
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --dataset Pascal_aug --model-zoo EncNet_Resnet101_COCO --aux --se-loss --lr 0.001 --syncbn --ngpus 4 --checkname res101 --ft
# Finetuning on original set
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --dataset Pascal_voc --model encnet --aux  --se-loss --backbone resnet101 --lr 0.0001 --syncbn --ngpus 4 --checkname res101 --resume runs/Pascal_aug/encnet/res101/checkpoint.params --ft

when I use the first command, the system point out no parameters about --syncbn and --ngpus。just to delete the --syncbn and --ngpus ，the command can work。Meanwhile, I can not find these two parameters in the train.py。

So, are these two parameters necessary? In other words, do I need to modify the contents of the train.py files to use them? Or is it just because the cmd link content has not been updated? I just need to simply delete these two parameters?

**Comment by talotta-anselmo** [comment]
Is there a way to disable syncbn? 
It makes training fail on my local machine, with no GPU.
It makes it impossible for me to debug training code.
Thanks.

**Comment by zhanghang1989** [comment]
Simply changing this to regular BN https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/experiments/segmentation/train.py#L150
